### ISSUE TITLE
Does not continue

### ISSUE THREAD

**Issue by rocuh** [techl/MonoRemoteDebugger#1]
I was finding continue was not working - it would step, disabling the step once it has been handled seemed to work ok:

in DebuggedMonoProcess.cs

```
        private void HandleStep(StepEvent stepEvent)
        {
            _engine.Events.StepCompleted(_mainThread);
            logger.Trace("Stepping: {0}:{1}", stepEvent.Method.Name, stepEvent.Location);

            isStepping = false;

            if (currentStepRequest != null)
                currentStepRequest.Disable();
        }
```

**Comment by techcap** [comment]
Thank you very much! Fixed and released v1.0.1

**Comment by techcap** [comment]
Updated v1.0.2. Now it supports Visual Studio Output Window.
